### PR TITLE
Migrate labels with dependencies solved

### PR DESCRIPTION
**Pull request by Hosuke**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

File: models/labels/addresses/__single_category_labels__/balancer_v1/labels_balancer_v1_pools_ethereum.sql [hunk 1]
```diff
@@ -13,46 +14,55 @@
...
+        AND call_block_time >= date_trunc('day', now() - interval '7' day)
     {% endif %}
 ),
 
 state_with_gaps AS (
-    SELECT events.block_number, events.pool, events.token, CAST(events.denorm AS double),
+    SELECT events.block_number
+           , events.pool
+           , events.token
+           , CAST(events.denorm AS double) AS denorm,
```
Comment (jeff-dude) [inline models/labels/addresses/__single_category_labels__/balancer_v1/labels_balancer_v1_pools_ethereum.sql:58]: on line 46, you cast to `uint256`, but do `double` here -- do we know what we want the output to be?

File: models/labels/addresses/__single_category_labels__/balancer_v2/labels_balancer_v2_gauges.sql [hunk 2]
```diff
@@ -1,4 +1,6 @@
-{{config(alias = alias('balancer_v2_gauges'),
+{{config(
+        tags=['prod_exclude', 'dunesql'],
```
Comment (jeff-dude) [inline models/labels/addresses/__single_category_labels__/balancer_v2/labels_balancer_v2_gauges.sql:2]: remind me, why `prod_exclude` tag, when it's not on spark version?

Comment (jeff-dude) [reply models/labels/addresses/__single_category_labels__/balancer_v2/labels_balancer_v2_gauges.sql:2]: this applies to all upstream spells in here too

Comment (Hosuke) [reply models/labels/addresses/__single_category_labels__/balancer_v2/labels_balancer_v2_gauges.sql:2]: Sorry at the time I translated these spells, they are `prod_exclude` in spark spells, due to invalid sources.
I will update the tag

File: models/labels/addresses/__single_category_labels__/balancer_v1/labels_balancer_v1_pools_ethereum.sql (hunk 1 above)
Comment (Hosuke) [reply models/labels/addresses/__single_category_labels__/balancer_v1/labels_balancer_v1_pools_ethereum.sql]: I am not sure with this field actually.
I have casted the denorm into `uint256` type.

`denorm` is `uint256` type in other decoded tables:
https://dune.com/queries?blockchain=ethereum&category=decoded_project&namespace=balancer_v1&contract=BPool&blockchains=ethereum&abi=rebind

Comment (jeff-dude) [reply models/labels/addresses/__single_category_labels__/balancer_v1/labels_balancer_v1_pools_ethereum.sql]: nice, then this is the right call

**Review by jeff-dude — APPROVED** [review]
thank you as always 👑

**Pull request merged**